src/interfaces/stepper-interfaces/PeopleAndRoles/org-person-interface.ts

```diff
@@ -15,12 +15,39 @@ export interface ApiPersonIF {
 
 export interface OrgPersonIF {
   officer: ApiPersonIF
-  role?: RoleTypes
-  roles: RoleIF[] | any
```

Comment:
I got rid of this "role" vs "roles" overload. Now, when directors are fetched, they are converted into THIS object structure.

(It's too bad the API returns different object schemas for the parties vs directors endpoints.)